media/index.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: media
+title: Docs
+---          
+<!-- begin Documents -->
+<hr class="invisible">
+<div id="media-docs">
+
+<div class="small-12 medium-10 medium-offset-1 columns">
+    <div class="row text-center">
+        <h2>Docs</h2>
+    </div>    
+    <table class="files-listing">
+        <thead>
+            <tr>
+                <th>File Name</th>
+                <th width="250">Last modified</th>
+                <th width="150">Size</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><i class="fa fa-file-pdf-o"></i><a href="https://downloads.openmicroscopy.org/presentations/2017/Cambridge-Training/walkthrough_day_1.pdf">walkthrough_day_1.pdf</a> <span data-tooltip aria-haspopup="true" class="has-tip top" data-disable-hover="false" tabindex="2" title="Day 1 - example workflows for managing images; viewing, analyzing and publishing images using OMERO plugins; filtering, searching and analyzing using Fiji features; and publishing with OMERO.figure.">(description)</span></td>
```

Comment:
If we are listing the filenames only, my main concern are the generic names and the confusion with future entries. Two options would be:
- prefix each presentation with the date/event but this is long creates some redundancy in the downloads URL
- group items by categories e.g. `2017 Cambridge Training`. Has such a layout been discussed and agreed?

Reply:
I vote for and pushed the latter for the reasons you listed in the former.